config/cmake/HDFMacros.cmake

```diff
@@ -458,6 +458,21 @@ macro (HDF_DIR_PATHS package_prefix)
   endif ()
   message(STATUS "Final: ${${package_prefix}_INSTALL_DOC_DIR}")
 
+  # Define the needed INSTALL_RPATH for HDF Standard binary packages
+  option (HDF5_SET_DEFAULT_RPATH "Set the default RPATH for the common installation" OFF)
```

Comment:
Why again with even more options? Better solutions were given in the related issue in the first place, including never overriding `CMAKE_INSTALL_RPATH`.

Reply:
Because it would require multiple options with STRING attributes to be on the command line when one option just does it correctly.

Reply:
I get the convenience aspect, but really the command-line options should pretty much be the standard way. Plus, if you turn this option on we're back to overriding what may have been desired to be passed in from the user, in addition to what we're setting.

Are we really sure that modifying the rpath is absolutely necessary and that we aren't doing something wrong here in the first place?

Reply:
Yes, modifying the rpath gives the app/lib an expected search path